proposals/lip-0052.md

```diff
@@ -2295,7 +2295,23 @@ Let `genesisBlockAssetBytes` be the `data` bytes included in the block assets fo
 
 ### Endpoints for Off-Chain Services
 
-This section specifies the non-trivial or recommended endpoints of the NFT module and does not include all endpoints. TBA
+This section specifies the non-trivial or recommended endpoints of the NFT module and does not include all endpoints. 
+
+#### hasNFT
+
+This function checks whether an NFT is owned by a specific user.
```

Comment:
I would be slightly more specific and aligned with the implementation and say:
"This function checks whether an NFT exists and is owned by a specific user."

Reply:
Done.